src/store/actions/auth.js

```diff
@@ -0,0 +1,87 @@
+import * as actionTypes from './actionTypes';
+import axios from '../../posts-axios';
+
+export const authStart = () => {
+    return {
+        type: actionTypes.AUTH_START
+    };
+};
+
+export const authSuccess = (token, userId) => {
+    return {
+        type: actionTypes.AUTH_SUCCESS,
+        idToken: token,
+        userId: userId
+    };
+};
+
+export const authFail = (error) => {
+    return {
+        type: actionTypes.AUTH_FAIL,
+        error: error
+    };
+};
+
+export  const logout = () => {
+    localStorage.removeItem('token');
+    localStorage.removeItem('expirationDate');
+    localStorage.removeItem('userId');
```

Comment:
This logic should be handled in the reducer NOT in the action

Reply:
In the course he handle it in the action.
How we can handle it in the reducer?